journey-map/public/index.mjs

```diff
@@ -84,6 +91,7 @@ const loadJourneyMaps = async () => {
         const journeyResponse = await fetch(`./${encodeURIComponent(upperToKebab(journeyType))}.yaml`);
         journeyMaps[journeyType] = yaml.parse(await journeyResponse.text());
     }));
+
     const nestedResponse = await fetch('./nested-journey-definitions.yaml');
     nestedJourneys = yaml.parse(await nestedResponse.text());
```

Comment:
I did also try and separate the nested journeys into separate files instead (there's another ticket to extract a KBV nested journey so thought it might be worth doing to avoid a long file) but I wasn't sure if it was within the scope of this ticket to also update the states machine to look for these separate files instead or if that should be part of another ticket?

Reply:
Good question - feels like it should definitely be a separate PR, although I'm happy for you to do it as part of this ticket or a separate one as you prefer

Reply:
I think I'll do it as part of [7035](https://govukverify.atlassian.net/browse/PYIC-7035) to not block this one